torch/utils/cpp_extension.py

```diff
@@ -1375,7 +1375,7 @@ def _get_cuda_arch_flags(cflags=None):
     ])
```

Comment:
```suggestion
        ('Ampere', '8.0+PTX'),
    ])
```

Reply:
Change applied. Thanks for pointing out!